openshift/ansible/__init__.py

```diff
@@ -35,15 +40,25 @@ def __str__(self):
 
 
 class OpenShiftAnsibleModule(AnsibleModule):
+    """
+    Abstraction of an OpenShift Ansible Module.
+    """
+
     def __init__(self, openshift_type):
+        """
+        Creates an instance of the OpenShiftAnsibleModule.
+
+        :parameters openshift_type:
+        :type openshift_type:
+        """
         self.openshift_types = self._init_types()
 
-        if openshift_type not in self.openshift_types:
+        if openshift_type not in list(self.openshift_types.keys()):
```

Comment:
Interested in knowing more about the reason behind this change, I thought in was the pythonic way to check for the existence of a key.